DueTimer.cpp

```diff
@@ -160,7 +169,7 @@ uint8_t DueTimer::bestClock(double frequency, uint32_t& retRC){
 	float bestError = 9.999e99;
 	do
 	{
-		ticks = (float) VARIANT_MCK / frequency / (float) clockConfig[clkId].divisor;
+		ticks = (float) SystemCoreClock / frequency / (float) clockConfig[clkId].divisor;
```

Comment:
This will probably be an issue for ArduinoDue boards. Can you point the definition in headers for this in both ArduinoDue and OpenTracker?